src/runtime/dom-particle-base.ts

```diff
@@ -143,7 +143,7 @@ export class DomParticleBase extends Particle {
     return [];
   }
 
-  forceRenderTemplate(slotName: string) {
+  forceRenderTemplate(slotName: string = '') {
```

Comment:
does it make sense to force a render template for an empty slot name?

Reply:
Afaict, the only place this is actually called is in `multiplexer-dom-particle.ts` which does
```
      this.forceRenderTemplate();
```

Reply:
CC @mmandlis

Reply:
I think that this would be a good candidate for an optional type. 
We can either define this with generics like so: 
```
type Either<L, R>  = L | R;
type Optional<T> = Either<void, T>;
```
which can be used as `slotName: Optional<string>`.

A quicker way (that doesn't introduce a constructed type throughout the codebase) it to just write: `slotName: string | void`